lib/events.js

```
@@ -1,68 +1,33 @@
-			AntiViewOnce(msg),
-			updateGroupMetadata(msg),
-			MessagesDebug(msg),
-			StickerCMD(msg)
-		);
-	}
-	if (msg.isGroup) {
-		if (isJidGroup(msg.from) && configValues.disablegc) return; // Ignore if disablegc is true in a group
-		tasks.push(Antilink(msg), schedules(msg), AntiWord(msg), AutoKick(msg));
-	}
-	await Promise.all(tasks);
+  let tasks = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'tasks' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)